tests/test_rpc.c

```diff
@@ -28,100 +28,325 @@
 #include <stdlib.h>
 #include <stdarg.h>
 
+#include <libnetconf2/messages_client.h>
+#include <libnetconf2/netconf.h>
+#include <libyang/printer_data.h>
+#include <libyang/tree_data.h>
```

Comment:
All includes redundant, all `libnetconf2` client headers are in `nc_client.h` and all `libyang` headers in `libyang/libyang.h`.